vms/platformvm/txs/executor/helpers_test.go

```diff
@@ -215,6 +215,8 @@ func addSubnet(
 
 	stateDiff.AddTx(testSubnet1, status.Committed)
 	require.NoError(stateDiff.Apply(env.state))
+	require.NoError(env.state.Commit())
+	defaultBalance -= env.config.GetCreateSubnetTxFee(env.clk.Time())
```

Comment:
Is there _any_ way to avoid doing this? This really breaks a lot of assumptions around the expectation of `defaultBalance` imo

Reply:
Dropped it. It's not strictly necessary rn, since the only test making asserting something related to subnet creation tx is pre banff.
I have a thread cleaning up UTs (https://github.com/ava-labs/avalanchego/issues/2453). I'll look for a solution that does not alter defaultBalance there (possibly using a key just for subnet creation)